hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1507,64 +1507,79 @@
-            alert propagates to an NMI.
+            - Similar to chip_otbn_rnd_entropy, but verifies the URND bits.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: chip_otbn_reg_uncorrectable_alert
-      desc: '''Verify the reg uncorrectable alert from OTBN.
+      name: chip_otbn_idle
+      desc: '''Verify the OTBN idle signal to clkmgr.
 
-            Details TBD. Ensure that this alert propagates to an NMI.
+            - Write the OTBN clk hint to 1 within clkmgr to indicate OTBN clk is ready to be gated.
+              Verify that the OTBN clk hint status within clkmgr reads 0 (OTBN is idle).
+            - Start an OTBN operation.
+              Verify that the OTBN clk hint status within clkmgr now reads 1 (OTBN is not idle),
+              before the operation is complete.
+            - After the OTBN operation is complete, check the output for correctness.
+              Verify that the OTBN clk hint status within clkmgr now reads 0 again (OTBN is idle),
+              after the operation is complete.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: chip_otbn_mem_encr
-      desc: '''Verify the encryption of the mem within OTBN using the key provided by the OTP.
+      name: chip_otbn_edn_reset
+      desc: '''Verify that the EDN clock / reset is connected to OTBN.
 
-            Details TBD.
+            - Use connectivity assertion check to verify that the right clock and reset signals are
+              connected for receiving the entropy data from EDN.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: chip_otbn_idle
-      desc: '''Verify the OTBN idle signal to clkmgr.
+      name: chip_otbn_ast_ram_cfg
+      desc: '''Verify that the ram_cfg signal from AST is connected to OTBN.
 
-            - Write the OTBN clk hint to 1 within clkmgr to indicate OTBN clk is ready to be gated.
-              Verify that the OTBN clk hint status within clkmgr reads 0 (OTBN is idle).
-            - Initiate an OTBN operation with a known key, plain text and digest.
-              Verify that the OTBN clk hint status within clkmgr now reads 1 (OTBN is not idle),
-              before the OTBN operation is complete.
-            - After the OTBN operation is complete, verify the digest for correctness.
-              Verify that the OTBN clk hint status within clkmgr now reads 0 again (OTBN is idle),
-              after the OTBN operation is complete.
+            - In open source, this is verified by a simple connectivity assertion check.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_otbn_mem_scramble
+      desc: '''Verify the OTBN can receive keys from the OTP to scramble the OTBN imem and dmem.
+
+            - Have OTBN fetch a new key and nonce from the OTP_CTRL.
+            - Write and read-check OTBN and IMEM for consistency.
```

Comment:
I'm not convinced this second step is really needed: if we'd messed up our memory scrambling, all of our block-level tests would be broken!

Reply:
Yes, but at the block level, you provide (likely randomize) the keys. At the chip level, they come from OTP ctrl. The second step is very much needed to prove the connections are there.

Reply:
Right, makes sense.